appSettings.json

```diff
@@ -32,7 +32,9 @@
         "startOperationId": 1,
         "stopOperationId": 1001,
         "totalNetVectorsToIngest": 3000000
-      }
+      },
+      "enableShardedDiskANN": false,

```

Comment:
make a subsection like streaming for filter and put everything inside that? I expect there will be more in future here.